### DISCUSSION TITLE
Importing multiple instances from FBX doesn't work

### DISCUSSION THREAD

**Discussion by maximesouvestre** [stride3d/stride#1402] [Q&A] [answered]
Hello,
I'm trying to import an FBX that contains instance of an object into Stride, but it doesn't seem to work.

Here is what my 3D scene looks like in Blender. Simple scene with two instances and one master (which is hidden). I then export it as an FBX.
![fbx_instance_Blender](https://user-images.githubusercontent.com/50681359/162803514-14792bbf-5a7d-412d-a765-4d54b22912e4.png)

When I import the file into Stride, I only see the first instance of all my instances:
![fbx_instance_Stride](https://user-images.githubusercontent.com/50681359/162803624-773cee8a-e16b-408a-be61-12ef8e9f5b1d.png)

I checked my file with both the 3D viewer from Windows and Maya and the results are correct in those two softwares (I can see all my instances). So the issue is not related to my file I would say.
![fbx_instance_3Dviewer](https://user-images.githubusercontent.com/50681359/162803731-34d64762-e7ee-4c58-8461-5b6a82a83767.png)
![fbx_instance_Maya](https://user-images.githubusercontent.com/50681359/162803743-71b1e233-ad4d-401a-93f8-6c80916f4517.png)

So now I wonder: does Stride support importing instance from an FBX file?

Thank you!

**Comment by tebjan** [comment, marked as answer]
Hello, I believe that isn't implemented, looks like multiple instances and multiple animations aren't supported. can you export the .fbx differently so that the meshes are all "baked"? otherwise, you might need a tool similar to this: https://github.com/JackPilley/BlenderToStrideAnimationSeparator
